galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -256,30 +256,6 @@ messages:
       - name: problem
         type: string
 
-  - name: "NamespaceMultipleInjectionLabels"
```

Comment:
This one still seems useful since we didn't merge my PR to allow `istio.io/rev=default`?

Reply:
Do we plan to merge it before testing day?  If not, then I need to re-do this PR.